src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -121,7 +121,7 @@
     private String jenkinsTunnel;
     @CheckForNull
     private String credentialsId;
-    private int containerCap = Integer.MAX_VALUE;
```

Comment:
Was this not setting the default value properly? AFAICT it hasn't defaulted to `0` since [5 years ago](https://github.com/jenkinsci/kubernetes-plugin/blame/e2dc95005ddf3d456de94e1a202e6f2e9930a221/src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java#L106).

Reply:
Instance initializers are not run during deserialization, so the field would have been left at its Java primitive default of 0.

Reply:
I understand that the deserialized value won't be (re)set to `Integer.MAX_VALUE`, but wasn't it serialized as `Integer.MAX_VALUE` in the first place? At least for the last several years. If it was serialized with a non-zero value then IIUC it should deserialize with the same value.

Reply:
Sure. I have no idea what issue is being addressed by this PR, just reviewing.